app/models/concerns/digital_object_concerns/digital_object_data/setters/projects.rb

```diff
@@ -5,7 +5,7 @@ module Projects
 
       def set_projects(digital_object_data)
         return unless digital_object_data.key?('projects')
-        self.projects = Set.new(digital_object_data['projects'].each { |digital_object_data_project|
+        self.projects = Set.new(digital_object_data['projects'].map { |digital_object_data_project|
```

Comment:
this seems like an error you fixed, but the result will be that projects is an array instead of the last resolved key from the digital object data. that sounds right to me, and the property has a plural name, but I just want to flag in case we use the projects attribute as if it were singular.

Reply:
I think we are always expecting a Set and we treat the field like an array (for the most part).